### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by DianaBeki**
The following was done in this task
- completed contact form section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -284,10 +284,10 @@ <h3 class="langTitle">Skills</h3>
...
     <footer class="footer">
       <div id="footer-container">
+        <h2 class="title">Contact me</h2>
+        <p class="paragraph-1">If you have an application you are interested in developing, a feature that you need built
+            or a project that needs coding. I’d love to help with it</p>
         <form class="form" action="https://formspree.io/f/xpznoaqp" method="POST">
-            <h2 class="title">Contact me</h2>
-            <p class="paragraph-1">If you have an application you are interested in developing, a feature that you need built
-                or a project that needs coding. I’d love to help with it</p>
             <ul class="footer-list">
```
Comment (vickymarz) [inline index.html:291]: - [ ] KIndly increase the width of your textarea and input field to be on the same line as the contact section text description to match the Figma design 👍🏽 

Figma Design 👇🏽 
image

Your Design 👇🏽 
image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @DianaBeki, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -344,50 +353,53 @@ h4 {
...
-  height: 112px;
   padding: 0 24px;
   border: 1px solid #cfd8dc;
   border-radius: 8px;
-  margin: 0 16px;
   font-family: "Poppins", serif;
   outline: none;
+  font-size: 17px;
+  height: 170px;
+  color: #b3bac5;
```
Comment (ichala) [inline style.css:394]: - [ ] as mentioned by the previous reviewer kindly increase the width of the textarea to match the other inputs

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @DianaBeki 
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Good Design✔️ 
- No linters errors✔️

 ## Required Changes ♻️

- [ ] 🛑 **_Check the comments below_**

## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**